core/test-runner.ts

```diff
@@ -18,6 +19,16 @@ export class TestRunner {
    private _testResults = new TestSetResults();
    private _currentTestResults: TestResults;
    private _currentTestFixtureResults: TestFixtureResults;
+   private _output: TestOutput;
+
+   constructor (output?: TestOutput) {
+       // If we were given a TestOutput, use it, otherwise make one
+       if (output !== undefined) {
```

Comment:
Cool, so for example they would call new TestRunner() instead of createTestRunner() which could be a factory we provide them - I guess new TestRunner just makes more sense eh?